docs/current_docs/quickstart/428201-custom-function.mdx

```diff
@@ -14,14 +14,14 @@ So far, you've been calling individual Dagger Functions using the Dagger CLI. Bu
 
 One option is, of course, to stitch together calls using the CLI and a shell script or a Makefile However, this approach is not recommended, because you will typically end up with long and unwieldy CLI commands and shell scripting "glue" that is hard to debug and maintain.
```

Comment:
> , of course,

I think this style makes sense in conversation but feels cumbersome to read.

Reply:
Fixed